code/game/objects/items/food/snacks.dm

```diff
@@ -206,14 +207,20 @@
 	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/bbqsauce = 1)
 	tastes = list("peanuts" = 3, "bbq sauce" = 1, "arguments" = 1)
 
+/obj/item/food/peanuts/banappeal
+	name = "\improper Gallery's Ban Appel roast peanuts"
+	desc = "Yearly lobbying is denied not because the roasting apples are toxic, but because they keep evading the ban."
+	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/toxin/cyanide = 1) //uses roasted poison apples
```

Comment:
what

Reply:
Its contraband only, but I could make it just toxin. Cyanide is appropriate because apples, and it's only 1u

Reply:
I meant what like the wordplay is a bit mystifying and the cyanide choice is also perplexing

Reply:
Apples contain trace cyanide in real life. It's also a throwback to poison apples.

The description is a play on words describing Ban Appeal forum threads. Because the brand is "Gallery's Peanuts" the joke alludes to the peanut posting threads that inevitably follow dumb appeals. The exact description I wrote conveys the joke without being a block of text.

I'm not sure why I had to explain the joke!

Reply:
The name of the item is literally "The peanut gallery roasts a ban appeal thread" by the way

Reply:
Not that it really matters, but the apple thing has me confused. They're roasted peanuts. How do roasted apples come into it? Why not just have the item name as "The Peanut Gallery's roasted peanuts", the brand as The Peanut Gallery, and remove the apple thing entirely? The toxicity and the cyanide could stay, and the description could just allude to the peanuts being toxic. But I don't see why people would ever buy food that poisoned them. Occasionally one person who knew how to use the cyanide well would take advantage of it but once everyone knew it was stupid to eat them you'd almost never see them.

Reply:
Maybe they roast the peanuts in apple juice or something, I don't know how peanuts are roasted.

"Gallery's peanuts" is already how the other peanuts are named, the joke needs to keep the consistency.

It's contraband, it doesn't have to be healthy to eat (see: the Bronx Paradise bar). Cyanide could be useful for slime characters, or for inventive traitors, so this isn't an issue.

Reply:
I'm telling you it doesn't make sense until you explain it but whatever

Reply:
maybe just for you, nobody else seems to mind.

Reply:
* Avoid `nospacescase`. Use `ban_appeal` or something like that.
* The apple/cyanide thing doesn't make sense to me either. They're peanuts. Maybe try a different (weak) toxin instead, unless if you're going to make these actually apples or apple-related.
* The joke itself is fine, it can pass as an IC thing with these peanuts being toxic and thus banned from consumption.